tox.ini

```diff
@@ -69,7 +69,7 @@ commands =
 [testenv:integration-charm]
 description = Run charm integration tests
 deps =
-    juju==2.9.11 # juju 3.3.0 has issues with retrieving action results
+    juju==2.9.38.1 # juju 3.3.0 has issues with retrieving action results
```

Comment:
I think the comment "# juju 3.3.0 has issues with retrieving action results" makes no sense anymore and should be removed... moveover 3.3.0 is wrong and never existed. It was a comment about 3.0.0 initially.

Reply:
jfyi: the comment referred to the juju agent version while there is also the libjuju package which is confusingly named `juju`